packages/react/__tests__/compat/server.js

```diff
@@ -41,7 +41,7 @@ test('extracted rules have correct keys when dealing with multiple caches', () =
   )
 
   let cache2 = createCache({ key: 'ssr-second-key' })
-  let { extractCritical: extractCritical2 } = createEmotionServer(cache2)
+  let { extractCritical: extractCriticalToChunks } = createEmotionServer(cache2)
```

Comment:
in here it actually should be kept as `extractCritical2` :)

Reply:
Good catch :)))